src/content/app/tools/blast/state/blast-results/blastResultsSlice.ts

```diff
@@ -52,7 +52,7 @@ export type OptionalSubmissionParameters = Partial<
 >;
 export type BlastSubmissionParameters = MandatorySubmissionParameters &
   OptionalSubmissionParameters & {
-    title: string;
+    jobName: string;
```

Comment:
Woah, woah. Wait a second. The name of the parameter that gets submitted to EBI BLAST is `title`. Since EBI BLAST only has a concept of job (and not of our submission), the `title` field will be applied to every job.

- The `parameters` field of the payload (see blastApiSlice) should only contain valid BLAST parameters. `jobName` isn't one.
- We can either save submission name at the top level of a submission, or in the `title` field of submission parameters. Not sure which to prefer, but slightly leaning toward the `title` parameter. In either case, it shouldn't be called _job_ name

Reply:
i found title confusing and in the frontend code we were calling it jobName which is better even though it is submission name. i will look into this maybe next week now.

Reply:
For reference: here's where `title` comes from:

https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?pageId=94147939#NCBIBLAST+HelpandDocumentation-RESTAPI

![image](https://user-images.githubusercontent.com/6834224/195887599-9299ba3f-5325-4f0d-aa6e-b7cf0e9984b0.png)

But again, for EBI, it's the title of a job, rather than of a batch of jobs, like it is for us. However, we can still make use of this field for our purposes.

Reply:
was that agreed with Backend (toolsApi) that the submission name will be the same title thats required from EBI. Also are we happy with the name `title` in our code. should it not be something like `submissionTitle` | `submissionName` or keep it just `title` to make it match whats required from the backend?

Reply:
The code has been refactored to rename jobName to submissionName and moved it outside parameters and as agreed we are not going to use `title` thats provided by the EBI backend since `title` is specific for how EBI tools work which is associating a title to each submitted job whereas submissionName is very specific to our FrontEnd UI where it is the name of each submission which can include multiple jobs.